thirdeye/thirdeye-pinot/src/main/java/com/linkedin/thirdeye/dashboard/resources/AnomalyResource.java

```diff
@@ -240,7 +242,18 @@ public Response createAnomalyFunction(@NotNull @QueryParam("dataset") String dat
     anomalyFunctionSpec.setBucketSize(dataGranularity.getSize());
     anomalyFunctionSpec.setBucketUnit(dataGranularity.getUnit());
 
-    anomalyFunctionSpec.setExploreDimensions(exploreDimensions);
+    // Ensure that the explore dimension names are ordered as schema dimension names
```

Comment:
I will work on a better solution that does not need the dimensions to be restricted in a certain order.